config/samples/kustomization.yaml

```diff
@@ -0,0 +1,59 @@
+resources:
```

Comment:
Oh, I misunderstood your instructions about this file - I think it makes more sense to have it committed rather than generated each time.